examples/resources/nios_ipam_ipv6networktemplate/resource.tf

```
@@ -0,0 +1,52 @@
+// Manage an IPAM IPv6 Network Template with Basic Fields
+resource "nios_ipam_ipv6networktemplate" "ipam_ipv6_network_template_basic" {
+  name = "example-network-template"
```

```suggestion
  name = "example-ipv6-network-template"
```
applies to all instances. Please use the same name in data source example as well